src/app/onboarding/legal-credentials/page.tsx

```diff
@@ -51,12 +38,12 @@ export default function Page() {
   useScrollToTop();
 
   // initialize form with data from onboarding context
-  const form = useForm<z.infer<typeof legalExperienceSchema>>({
-    resolver: zodResolver(legalExperienceSchema),
+  const form = useForm<z.infer<typeof legalCredentialSchema>>({
+    resolver: zodResolver(legalCredentialSchema),
     defaultValues: {
-      stateBarred: onboarding.profile.state_barred,
-      barNumber: onboarding.profile.bar_number,
-      eoirRegistered: onboarding.profile.eoir_registered,
+      stateBarred: onboarding.profile.state_barred ?? undefined,
```

Comment:
Did we do this to ensure that it's not null?

Reply:
yer